pombola/south_africa/views/geolocalization.py

```diff
@@ -0,0 +1,309 @@
+import mapit
+import requests
+
+from .constants import API_REQUESTS_TIMEOUT
+
+from django import forms
+from django.contrib.gis.geos import Point
+from django.contrib.gis.measure import D
+from django.core.urlresolvers import reverse
+from django.http import Http404
+from django.shortcuts import redirect
+from django.utils.http import urlquote
+
+from haystack.forms import SearchForm
+
+from pombola.core import models
+from pombola.core.views import (
+    BasePlaceDetailView, PlaceDetailView, PlaceDetailSub)
+from pombola.search.views import GeocoderView
+
+from pombola.south_africa.models import ZAPlace
+
+# In the short term, until we have a list of constituency offices and
+# addresses from DA, let's bundle these together.
+CONSTITUENCY_OFFICE_PLACE_KIND_SLUGS = (
+    'constituency-office',
+    'constituency-area',  # specific to DA party
```

Comment:
If I remove the space I get the flake8 error:
`E262 - Inline comment should start with '# '`
so I'm going to leave this one unchanged if that is OK.